test/course_bot/internationalization_test.clj

```diff
@@ -0,0 +1,32 @@
+(ns course-bot.internationalization-test
+(:require [clojure.test :refer [deftest testing is]])
+(:require [course-bot.internationalization :as i18n :refer [normalize-yes-no-text]]))
+
+(deftest test-normalize-yes-no-text-en
+  (testing "valid yes input: lowercase en"
+    (is (= "yes" (normalize-yes-no-text "yes"))))
+
+  (testing "valid yes input: mixed-case en"
+    (is (= "yes" (normalize-yes-no-text "YeS"))))
+  
+  (testing "valid no input: lowercase en"
+    (is (= "no" (normalize-yes-no-text "no"))))
+
+  (testing "valid no input: mixed-case en"
+    (is (= "no" (normalize-yes-no-text "NO"))))
+
+  (testing "invalid input"
+    (is (= "Invalid input" (normalize-yes-no-text "some incorrect input")))))
+
+(deftest test-normalize-yes-no-text-ru
+  (testing "valid yes input: lowercase ru"
+    (is (= "yes" (normalize-yes-no-text "да"))))
```

Comment:
Looks like it will be good to use `are` here.

Text descriptions can be moved inside `is` or removed.